integrationtests/upgrade_test.go

```diff
@@ -32,7 +32,7 @@ var _ = Describe("upgrade", func() {
 		})
 
 		It("upgrades many service instances", func() {
-			session := cf("upgrade-all-services", brokerName)
+			session := cf("upgrade-all-services", brokerName, "--instance-polling-interval", "10ms")
```

Comment:
question: I see you are setting different values in the tests for polling interval, but I fail to see if that serves any purpose. Is the polling interval being tested at all?